core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -2881,6 +2881,12 @@ public String toString() {
           .setScope(Scope.MASTER)
           .build();
 
+  public static final PropertyKey SECONDARY_MASTER_COMMON_SERVICES_EXCLUDED =
+      new Builder(Name.SECONDARY_MASTER_COMMON_SERVICES_EXCLUDED)
+          .setDefaultValue("METRICS_SINK,WEB,JVM_PAUSE_MONITOR")
+          .setDescription("Exclude start the services in secondary master common services.")
+          .setScope(Scope.SERVER)
+          .build();
```

Comment:
See comments about separating each of `METRICS_SINK`, `WEB`, and `JVM_PAUSE_MONITOR` into their own separately configurable services.

Also in general our existing `PropertyKey`s which toggle features typically use the `xxx.enabled=true|false` semantics. So it'd be preferable to maintain that.